### PR TITLE
Run Prettier on JS code fences, part 3

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Adding to https://github.com/mdn/content/pull/20713

The PR focuses only on JS code fences.

Idea is to gradually prettify all the JS code fences before the full automation.

### CODE REVIEW

File: files/en-us/web/api/background_tasks_api/index.md [hunk 1]
```diff
@@ -381,8 +389,8 @@ The function we'll be using as our task handler—that is, the function that wil
 function logTaskHandler(data) {
   log(`Running task #${currentTaskNumber}`);
 
-  for (i=0; i<data.count; i+=1) {
-    log(`${(i+1).toString()}. ${data.text}`);
+  for (i = 0; i < data.count; i += 1) {
```
Comment (teoli2003) [inline files/en-us/web/api/background_tasks_api/index.md:392]:
```suggestion
  for (let i = 0; i < data.count; i += 1) {
```

File: files/en-us/web/api/beforeunloadevent/index.md [hunk 2]
```diff
@@ -60,9 +60,10 @@ WebKit-derived browsers don't follow the spec for the dialog box. An almost-cros
 window.addEventListener("beforeunload", (e) => {
   const confirmationMessage = "\\o/";
 
-  (e || window.event).returnValue = confirmationMessage;     // Gecko + IE
-  return confirmationMessage;                                /* Safari, Chrome, and other
-                                                              * WebKit-derived browsers */
+  (e || window.event).returnValue = confirmationMessage; // Gecko + IE
+  return confirmationMessage;
+  /* Safari, Chrome, and other
+   * WebKit-derived browsers */
```
Comment (teoli2003) [inline files/en-us/web/api/beforeunloadevent/index.md:66]: These comments have to be moved to the right place.

File: files/en-us/web/api/biquadfilternode/getfrequencyresponse/index.md [hunk 3]
```diff
@@ -105,10 +105,14 @@ biquadFilter.gain.value = range.value;
...
-
-  for (i = 0; i <= myFrequencyArray.length-1;i++){
-    const listItem = document.createElement('li');
+  biquadFilter.getFrequencyResponse(
+    myFrequencyArray,
+    magResponseOutput,
+    phaseResponseOutput
+  );
+
+  for (i = 0; i <= myFrequencyArray.length - 1; i++) {
```
Comment (teoli2003) [inline files/en-us/web/api/biquadfilternode/getfrequencyresponse/index.md:114]:
```suggestion
  for (let i = 0; i <= myFrequencyArray.length - 1; i++) {
```

**Review by teoli2003 — DISMISSED** [review]

**Review by wbamberg — APPROVED** [review]
Nice work @OnkarRuikar ! Since @teoli2003 is out for a bit and it looks like you've addressed his comments, let's merge this :).

**Pull request merged**